src/components/button/_button.scss

```diff
@@ -8,165 +8,247 @@
+        color: $ui-05;
+      }
 
-    &:hover > .#{$prefix}--btn__icon,
-    &:focus > .#{$prefix}--btn__icon {
-      fill: $inverse-01;
+      &:hover > .#{$prefix}--btn__icon,
+      &:focus > .#{$prefix}--btn__icon {
+        fill: $inverse-01;
+      }
     }
-  }
 
-  .#{$prefix}--btn--ghost {
-    @include button-theme(transparent, transparent, $brand-01, $brand-01, $brand-01);
+    .#{$prefix}--btn--ghost {
+      @include button-theme(transparent, transparent, $brand-01, $brand-01, $brand-01);
 
-    &:hover,
-    &:focus {
-      color: $inverse-01;
+      &:hover,
+      &:focus {
+        color: $inverse-01;
+
+        .#{$prefix}--btn__icon {
+          fill: $inverse-01;
+        }
+      }
+
+      &:active {
+        color: $ui-05;
+      }
 
       .#{$prefix}--btn__icon {
-        fill: $inverse-01;
+        margin-left: $spacing-xs;
       }
-    }
 
-    &:active {
-      color: $ui-05;
+      &:hover:disabled,
+      &:focus:disabled {
+        color: $brand-01;
+
+        .#{$prefix}--btn__icon {
+          fill: $brand-01;
+        }
+      }
     }
 
-    .#{$prefix}--btn__icon {
-      margin-left: $spacing-xs;
+    .#{$prefix}--btn--danger {
+      @include button-theme(transparent, $support-01, $support-01, $support-01, $support-01);
+
+      &:hover {
+        color: $inverse-01;
+        border: $button-border-width solid transparent;
+      }
+
+      &:focus {
+        color: $inverse-01;
+      }
+
+      &:active {
+        color: $support-01;
+      }
+
+      &:hover:disabled,
+      &:focus:disabled {
+        color: $support-01;
+        border: $button-border-width solid $support-01;
+      }
+
+      &:hover > .#{$prefix}--btn__icon,
+      &:focus > .#{$prefix}--btn__icon {
+        fill: $inverse-01;
+      }
     }
 
-    &:hover:disabled,
-    &:focus:disabled {
-      color: $brand-01;
+    .#{$prefix}--btn--danger--primary {
+      @include button-theme($support-01, transparent, $inverse-01, darken($support-01, 10%), $ui-01);
 
-      .#{$prefix}--btn__icon {
-        fill: $brand-01;
+      &:hover:disabled,
+      &:focus:disabled {
+        color: $ui-01;
+        border: $button-border-width solid $support-01;
       }
     }
-  }
 
-  .#{$prefix}--btn--danger {
-    @include button-theme(transparent, $support-01, $support-01, $support-01, $support-01);
+    .#{$prefix}--btn--sm {
+      @include letter-spacing;
+      height: 2rem;
+      padding: $button-padding-sm;
+    }
 
-    &:hover {
-      color: $inverse-01;
-      border: $button-border-width solid transparent;
+    .#{$prefix}--btn--secondary + .#{$prefix}--btn--primary,
+    .#{$prefix}--btn--tertiary + .#{$prefix}--btn--danger--primary {
+      margin-left: $spacing-md;
     }
 
-    &:focus {
-      color: $inverse-01;
+    // Skeleton State
+    .#{$prefix}--btn.#{$prefix}--skeleton {
+      @include skeleton;
+      width: rem(150px);
     }
+  }
+  @include experimental('components') {
+    $button: #{$prefix}--btn;
```

Comment:
Everybody - Do y'all want to see this pattern in all our styles in our experimental blocks? I’m good with this, but I don’t want this pattern mixed with hand-writing (within our experimental blocks). Thanks!